<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>新宋体</family>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>AI_CarProgram</string>
  </property>
  <widget class="QLabel" name="label_video">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/spalsh/splash.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Face">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>90</y>
     <width>251</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>人脸点播</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Body">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>90</y>
     <width>251</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>人体跟随</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Voice">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>200</y>
     <width>251</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>语音助手：关闭</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Hide">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>200</y>
     <width>251</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>自动避障</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>530</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>10</y>
     <width>421</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>模式选择</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>330</y>
     <width>93</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>前</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>430</y>
     <width>93</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>左</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>430</y>
     <width>93</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>右</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>530</y>
     <width>93</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>后</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>430</y>
     <width>93</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>停</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>320</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>控制</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>421</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>实时视频流</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Image/Image.qrc"/>
 </resources>
 <connections/>
</ui>
